cmd/snap-seccomp/main.go

```diff
@@ -538,15 +591,26 @@ func parseLine(line string, secFilter *seccomp.ScmpFilter) error {
 			}
 		} else {
 			cmpOp = seccomp.CompareEqual
-			value, err = readNumber(arg)
+			value, err = readNumber(arg, syscallName)
 		}
 		if err != nil {
 			return fmt.Errorf("cannot parse token %q (line %q)", arg, line)
 		}
 
+		// For now only support EQ with negative args. If change this,
+		// be sure to adjust readNumber accordingly and use libseccomp
+		// carefully.
+		if syscallsWithNegArgsMaskHi32[syscallName] {
+			if cmpOp != seccomp.CompareEqual {
+				return fmt.Errorf("cannot parse token %q (line %q)", arg, line)
```

Comment:
Perhaps extend the error message to something like: `cannot parse token %q (line %q): unsupported comparison`?